packages/bot-web-ui/src/components/transaction.jsx

```diff
@@ -130,41 +134,62 @@ const Transaction = ({ active_transaction_id, contract, setActiveTransactionId }
         message={contract && <PopoverContent contract={contract} />}
     >
         <div
-            className='transactions__item'
+            className={classNames('transactions__item', {
+                'transactions__item-mobile': is_mobile,
+            })}
             onClick={contract && (() => setActiveTransactionId(contract.transaction_ids.buy))}
         >
-            {/* TODO: Re-enable when <Icon> is shared.
-        <div className='transactions__cell transactions__symbol'>
-            <TransactionIconWithText
-                icon={<UnderlyingIcon market={contract.underlying} />}
-                title={contract.display_name}
-            />
-        </div> */}
+            {/* TODO: Re-enable when <Icon> is shared. */}
             <div className='transactions__cell transactions__trade-type'>
-                {contract ? (
-                    <TransactionIconWithText
-                        icon={<IconTradeTypes type={contract.contract_type} />}
-                        title={getContractTypeName(contract)}
-                    />
-                ) : (
-                    <TransactionIconLoader />
-                )}
+                <div>
+                    {is_mobile &&
+                        (contract ? (
+                            <TransactionIconWithText
+                                icon={<Icon icon={`IcUnderlying${contract.underlying}`} />}
+                                title={'Underlying'}
+                            />
+                        ) : (
+                            <TransactionIconLoader is_mobile={is_mobile} />
+                        ))}
+                </div>
+                <div>
+                    {contract ? (
+                        <TransactionIconWithText
+                            icon={<IconTradeTypes type={contract.contract_type} />}
+                            title={getContractTypeName(contract)}
+                        />
+                    ) : (
+                        <TransactionIconLoader is_mobile={is_mobile} />
+                    )}
+                </div>
```

Comment:
We should have the same design in both desktop + responsive (respecting the responsive design).